<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientGameplayWindow</class>
 <widget class="QMainWindow" name="ClientGameplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CurveFewerClone</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gv_viewport">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>700</width>
      <height>600</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>276</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lab_nick1">
       <property name="text">
        <string>Player 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_score1">
       <property name="text">
        <string>Score: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_color1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_nick2">
       <property name="text">
        <string>No player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_score2">
       <property name="text">
        <string>Score: -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_color2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_nick3">
       <property name="text">
        <string>No player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_score3">
       <property name="text">
        <string>Score: -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_color3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_nick4">
       <property name="text">
        <string>No player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_score4">
       <property name="text">
        <string>Score: -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_color4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_disconnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_readiness">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_counter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
